### PR TITLE
feat: update libraries to fix vulns and publishing

### PR DESCRIPTION
**Pull request by KevinGhadyani-Okta**
### Description

Libraries needed updating in general, but that fixes some issues:
1. Vulnerability.
2. Lerna publish issue with `workspace:*` from Yarn v3.

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -3,5 +3,7 @@
   "npmClient": "yarn",
   "changelogPreset": "@okta/conventional-changelog-odyssey",
   "useWorkspaces": true,
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+  "useNx": false
```
Comment (kamronbatmanghelich-okta) [inline lerna.json:8]: This is false, but I see an nx.json. I don't know much about nx, can you give some insight into this?

Comment (KevinGhadyani-Okta) [reply lerna.json:8]: I should remove the nx.json.

File: scripts/setup.sh [hunk 2]
```diff
@@ -2,21 +2,14 @@
...
 
-echo "installing node ${NODE_VERSION}"
-if setup_service node $NODE_VERSION; then
-  echo "Installed node ${NODE_VERSION} successfully"
-else
-  echo "node ${NODE_VERSION} installation failed."
-fi
-
 # Note: Yarn will automatically switch over to yarn 3 after installing yarn 1.x
 YARN_VERSION=1.22.17
```
Comment (kamronbatmanghelich-okta) [inline scripts/setup.sh:6]: Update to 1.22.19

File: lerna.json (hunk 1 above)
Comment (KevinGhadyani-Okta) [reply lerna.json:8]: It looked like this, but it's gone now:
```json
{
  "tasksRunnerOptions": {
    "default": {
      "runner": "nx/tasks-runners/default",
      "options": {
        "cacheableOperations": ["build", "test", "svgo", "start"]
      }
    }
  },
  "targetDefaults": {
    "build": {
      "outputs": ["{projectRoot}/dist"]
    },
    "typecheck": {
      "dependsOn": ["^build"]
    }
  }
}
```

Comment (KevinGhadyani-Okta) [reply lerna.json:8]: 82c29667dd9b8b7f8ee27cccafc93c010ac31a38

File: scripts/setup.sh (hunk 2 above)
Comment (KevinGhadyani-Okta) [reply scripts/setup.sh:6]: ✅ 
f7d4bcb847005fa4641fff2ef3cffcf46b5e61a7

**Review by kamronbatmanghelich-okta — APPROVED** [review]

**Pull request merged**